content/applications/productivity/knowledge/management.rst

```diff
@@ -2,113 +2,240 @@
-permission.
+Alternatively, click :guilabel:`Search` in the top-left menu. Select an article and click
+:menuselection:`Actions --> Delete --> Delete` to remove the article permanently.
 
-.. image:: management/invite.png
-   :alt: pop-up window to invite users to access a Knowledge article
+- **Archive an article**
 
-Share online
-~~~~~~~~~~~~
+Click :guilabel:`Search`, select an article, and click :menuselection:`Actions --> Archive -->
+Archive`.
 
-To share articles **online**, activate the :guilabel:`Share to web` button. Doing so generates a URL
-link anyone can use to view the article.
+.. note::
+   - By default, archived articles have a 30-day expiration date before being permanently deleted.
+   - Archived articles are hidden in the search menu.
 
-Additionally, the share menu displays the default permission for internal members along with all the
-users who have been granted specific permission.
+.. _knowledge/management/export:
 
-.. image:: management/share-menu.png
-   :alt: Sharing menu with URL for online sharing and the list of members.
+Article export/import
+=====================
 
-.. _management/remove:
+Click :guilabel:`Search` to access the list of all articles. Select an article, then click
+:menuselection:`Actions --> Export` to :ref:`export <essentials/export_import_data/export-data>` it.
+Otherwise, click the :icon:`fa-cog` (:guilabel:`gear`) icon and :guilabel:`Import records` to
+:ref:`import <essentials/export_import_data/import-data>` an article.
 
-Removal
--------
+.. note::
+   - Only exported Knowledge articles are importable into Knowledge.
+   - Imported articles become hidden articles. Once retrieved, they appear in the
+     :guilabel:`Workspace` by default and retain their previously configured access rights.
 
-To remove an article, you can either **delete** it or **archive** it.
+.. _knowledge/management/retrieve:
 
-To delete an article, open it and click the vertical ellipsis button :menuselection:`(⋮) -->
-Delete`. The article is moved to the trash for 30 days before being permanently deleted. To restore
-it, click :guilabel:`Open the Trash`, select the article, and click :guilabel:`Restore`.
+Article retrieval
+=================
 
-To archive articles, click :guilabel:`Search`, select the article(s), and click
-:menuselection:`Action --> Archive --> Archive`. Archived articles are hidden from the search menu.
-To retrieve an archived article, add a custom filter to display them (:menuselection:`Search -->
-Filters --> Add Custom Filter`, and set :guilabel:`Active` as :guilabel:`is No`). Then, select the
-article(s) and go to :menuselection:`Action --> Unarchive`.
+Retrieving Knowledge articles consists of :ref:`restoring <knowledge/management/restore>` or
+:ref:`accessing <knowledge/management/access>` them from various Odoo apps.
 
-.. _management/structure:
+.. _knowledge/management/restore:
 
-Structure of articles
-=====================
+Restore articles
```

Comment:
I would merge this section with the previous one, saying how to delete, restore, and then archive, unarchive.